benchmark/ts2fs.py

```diff
@@ -0,0 +1,75 @@
+import tskit
+import numpy as np
+
+def _generate_fs_from_ts(ts, sample_sets=None):
+    """
+    """
+    def allele_counts(ts, sample_sets=None):
+        """
+        """
+        if sample_sets is None:
+            sample_sets = [ts.samples()]
+        def f(x):
+            return x
+        return ts.sample_count_stat(sample_sets, f, len(sample_sets),
+                                    span_normalise=False, windows='sites',
+                                    polarised=True, mode='site', strict=False)
+
+    mut_type = np.zeros(ts.num_sites)
+    for j, s in enumerate(ts.sites()):
+        mt = []
+        for m in s.mutations:
+            for md in m.metadata["mutation_list"]:
+                mt.append(md["mutation_type"])
+        if len(set(mt)) > 1:
+            mut_type[j] = 3
```

Comment:
When a scalar like 3 pops up like this, I would add a comment where it comes from.